vhdbuilder/packer/install-dependencies.sh

```diff
@@ -496,6 +496,66 @@ ls -ltr /usr/local/bin/* >> ${VHD_LOGS_FILEPATH}
 # shellcheck disable=SC2010
 ls -ltr /dev/* | grep sgx >>  ${VHD_LOGS_FILEPATH} 
 
+# populate the customized logrotate conf file for syslogs
+cat << EOF >> /etc/logrotate.d/aks-rsyslog
+/var/log/syslog
+{
+  rotate 4
+  daily
+  size 100M
+  missingok
+  notifempty
+  delaycompress
+  compress
+  postrotate
+      /usr/lib/rsyslog/rsyslog-rotate
+  endscript
+}
+
+/var/log/mail.info
+/var/log/mail.warn
+/var/log/mail.err
+/var/log/mail.log
+/var/log/daemon.log
+/var/log/kern.log
+/var/log/auth.log
+/var/log/user.log
+/var/log/lpr.log
+/var/log/cron.log
+/var/log/debug
+/var/log/messages
+/var/log/warn
+{
+  rotate 4
+  daily
+  size 25M
+  missingok
+  notifempty
+  compress
+  delaycompress
+  sharedscripts
+  postrotate
+      /usr/lib/rsyslog/rsyslog-rotate
+  endscript
+}
+EOF
+
+# populate the logrotate-aks unit and timer definitions
+cat << EOF >> /etc/systemd/system/logrotate-aks.service
```

Comment:
okay, it would be nice if these were in the packer json admittedly, a little unwieldy in bash